src/extension/content-script/onendwebln.js

```diff
@@ -41,7 +40,7 @@ async function init() {
       extractLightningData();
     }
     // forward account changed messaged to inpage script
-    else if (request.action === "accountChanged") {
+    else if (request.action === "accountChanged" && isEnabled) {
```

Comment:
The line below, can we replace "*"? we only want the inpage script to receive the message so it can emit the event for any listeners. Currently all frames on the website are receiving it.

Reply:
we can use window.location.origin here. which returns domain without any routes/ query parameters